packages/salesforcedx-utils-vscode/src/helpers/paths.ts

```diff
@@ -62,3 +64,24 @@ export function fileExtensionsMatch(sourceUri: vscode.Uri, targetExtension: stri
   const extension = sourceUri.path.split('.').pop()?.toLowerCase();
   return extension === targetExtension.toLowerCase();
 }
+
+function getSfdxDirectoryPath(): string {
+  return path.join(
+    getRootWorkspacePath(),
+    Global.SFDX_STATE_FOLDER
+  );
+}
+
+function getMetadataDirectoryPath(username: string): string {
+  return path.join(
+    getSfdxDirectoryPath(),
+    'orgs',
+    username,
+    'metadata'
+  );
+}
+
+export const projectPath = {
```

Comment:
Q: does this have to be a named export?  Could it be done like:

```
export {
  sfdxDirectoryPath,
  metadataDirectoryPath
}
```

so that when using these from the outside it would be:

```
import * from paths;

const sfdxDirPath = paths.sfdxDirectoryPath;
```

Reply:
hum...This is one of the situations where I don't know why I prefer namedExports.  I think it's b/c the only two states I'm aware of is named or default exports [doc](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export) I aways avoid default exports b/c it's easier to stay consistent with named exports. 

TBH I'm not even sure how it works if you export an object directly as suggested.  I'd have to test but I bet it just attaches each property as a stand along export we we'd end up with the reference vs value problem when stubbing in tests. 

bottom line I'd go with what we know and stick with the named export.

Reply:
Ok sounds good to me!  How about making the exported named object plural so it is `paths.projectPaths.getSfdxDirectoryPath()`